### PR TITLE
Feat/3 add title at the top of index.js file

### LINKED ISSUE #3: Add title for index.js file

### PR DESCRIPTION
**Pull request by ghani-othman**
#3

### CODE REVIEW

**Review by ghani-rmy — COMMENTED** [review]
> _**`LGTM`**_

**Review by ghani-rmy — CHANGES_REQUESTED** [review]
change

**Review by ghani-rmy — COMMENTED** [review]
correct it

File: routes/index.js [hunk 1]
```diff
@@ -1,3 +1,7 @@
+//
+// index.js file
+//
```
[the remark this answers is not in the document]
Comment (6A69696969692E) [reply routes/index.js:3]: resolve

**Review by ghani-rmy — APPROVED** [review]
> _**`LGTM`**_

**Pull request merged**